src/Tribe/Tickets.php

```diff
@@ -1514,12 +1537,13 @@ public static function get_event_attendees_count( $post_id, $args = [] ) {
 		 * @return array
 		 */
 		public static function get_all_event_tickets( $post_id ) {
-			$cache_key = self::$cache_key_prefix . $post_id;
-			$cache = new Tribe__Cache();
-			$tickets = $cache->get( $cache_key );
 
-			if ( is_array( $tickets ) ) {
-				return $tickets;
+			/** @var Tribe__Cache $cache */
+			$cache   = tribe( 'cache' );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 1 space but found 3 spaces

<sub>hash: 2b863865142fd14ad2eda6885f0f34a5</sub>